Jenkinsfile

```diff
@@ -115,6 +115,10 @@ pipeline {
               }
             }
             stage('Build and Package on Ubuntu Focal') {
+              when {
+                branch 'master'
+                beforeAgent true
+              }
```

Comment:
```suggestion
```

This should not be removed because we still want to test Focal packaging prior to merging PRs I thought?